### ISSUE TITLE
Ask for confirmation before creating duplicate data

### ISSUE THREAD

**Issue by mytskine** [inventaire/inventaire#300]
What happens know:

1. I want to add an author to a book
2. I type a part of his name, e.g. "Tom Tira".
3. I see no sensible completion (it seems only full words are searched, so the results will be completely wrong in this case).
4. I type the complete name and validate.

At step 4, if I don't wait for the search to complete and select an existing author in its suggestions, then I will create a duplicate author. This is extremely common. For example, the French Bande Dessinée author "Manu L." has a dozen of duplicate entries. Other types of items suffer from this.

I suggest that, on submission, the site should ask to explicitly reject the current entries and confirm the creation an homonym. It could be even simpler when there is a single homonym that has a wikidata ID (the default choice could be to accept this entry).

**Comment by mytskine** [comment]
I once again created a duplicate book, because the author had 3 instances and I knew only of one. Three authors are named "Jens Harder" but they are for the same real author. Each instance of this author had one book. I created a book under an author, but when I tried to add the edition, I saw it was already there.

I could find no way to merge authors, and when I tried to merge books (though the merging dialog did not make sense to me) I just obtained a message "api error". The complete lack of validation of inserts means the data is a complete mess, and it would be a huge task to fix years of bad data.

**Comment by maxlath** [comment]
I'm not sure to get what you mean by merging books, which dialog is that?

**Comment by jum-s** [comment]
this is correct. Merging authors is actually possible, but users need to have admins rights for that. We are thinking of opening up to more people to allow merging duplicates (there is actually a feature to find the best authors duplicate possibilities, also restrained to admins), but its not implemented yet (see issue #91 )

**Comment by maxlath** [comment]
oh, now I see which dialog you're talking about #380 sorry for that
